Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -316,11 +319,13 @@ void MainWindow::Pause()
 
 bool MainWindow::Stop()
 {
-  if (!Core::IsRunning())
+  if (!Core::IsRunning() || m_inquiry_in_progress)
```

Comment:
Not sure I understand the logic. If we are in the middle of asking the user to confirm the exit, shouldn't it return false instead of true?

Also, we should probably just make the confirm dialog a modal, so that we can't enter this function twice.